packages/babel-make-styles/config/tests.js

```diff
@@ -0,0 +1 @@
+require('@testing-library/jest-dom');
```

Comment:
this line was added by migration as well ?  meaning -> do we need jest-dom for this package ? (I'd assume we don't as its a node package)

Reply:
Or bad copy-paste 🤓 Anyway, replace it with a placeholder 👍